src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -36,6 +35,24 @@ object SlickDataAccess {
   implicit class StringToClob(val str: String) extends AnyVal {
     def toClob: Clob = new SerialClob(str.toCharArray)
   }
+
+  implicit class ConfigWithUniqueSchema(val config: Config) extends AnyVal {
```

Comment:
This function below (optionally) inserts a unique string into the config value `url`, that is seconds later used internally by slick to connect to the database.

When this code snippet activates for the default in memory database (see also reference.conf) this allows two things:

First, quick google searches weren't clear on a one-liner way to make slick behave like liquibase, only updating a schema. If special checks weren't added, each time the schema creation ran, after the first time, it would bomb out on "table already exists". There were a couple of search hits on how to peek in the database to see if the tables existed, but I ultimately backed off on this because...

Secondly, providing a clean, unique database each call to `DataAccess()`/`new SlickDataAccess()` leaves the behavior `SlickDataAccess` similar to the old `DummyDataAccess`. The coercion bug wasn't discovered earlier as `DummyDataAccess` was being used for tests. Now they can just use `DataAccess()` and a clean database will be created for the test.